protocol/bc/outputid.go

```diff
@@ -0,0 +1,60 @@
+package bc
+
+import (
+	"database/sql/driver"
+	"io"
+
+	"chain/crypto/sha3pool"
+	"chain/encoding/blockchain"
+)
+
+// OutputID identifies previous transaction output in transaction inputs.
+type OutputID Hash
```

Comment:
> I've also tried to use outid.Bytes() instead of tmp := outid.Hash; tmp[:] when I need a byteslice. Is that the right way?

I see nothing wrong with writing `outid.Hash[:]`, but I guess when the expression `outid` is actually a function call, it can't be written without a temporary variable. The `Bytes` method seems reasonable to me! But I wouldn't get dogmatic about using it everywhere; if `outid.Hash[:]` in one line is also an option, that's fine too IMO.